sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/implementation/directconnectivity/GatewayAddressCacheTest.java

```diff
@@ -418,20 +439,22 @@ public void tryGetAddresses_ForDataPartitions_Suboptimal_Refresh(
                                                                 null,
                                                                 null,
                                                                 ConnectionPolicy.getDefaultPolicy(),
-                                                                openConnectionsHandler);
+                                                                openConnectionsHandler,
+                                                                null
+                );
 
         String collectionRid = createdCollection.getResourceId();
 
         List<PartitionKeyRangeIdentity> pkriList = allPartitionKeyRangeIds.stream().map(
                 pkri -> new PartitionKeyRangeIdentity(collectionRid, pkri)).collect(Collectors.toList());
 
-        origCache.openConnectionsAndInitCaches(createdCollection, pkriList).blockLast();
+        origCache.resolveAddressesAndInitCaches(collectionLink, createdCollection, pkriList, AsyncDocumentClient.OpenConnectionAggressivenessHint.AGGRESSIVE).blockLast();
 
         assertThat(httpClientWrapper.capturedRequests).asList().hasSize(1);
         httpClientWrapper.capturedRequests.clear();
-        Mockito
-            .verify(openConnectionsHandler, Mockito.times(allPartitionKeyRangeIds.size()))
-            .openConnections(Mockito.any(), Mockito.any(), Mockito.any());
+//        Mockito
+//            .verify(openConnectionsHandler, Mockito.times(allPartitionKeyRangeIds.size()))
```

Comment:
why this one is being commented out?

Reply:
I am removing it since `resolveAddressesAndInitCaches` does not invoke open connections.